src/app/icd/client/CheckInHandler.cpp

```diff
@@ -127,6 +128,9 @@ CHIP_ERROR CheckInHandler::OnMessageReceived(Messaging::ExchangeContext * ec, co
     else
     {
         mpCheckInDelegate->OnCheckInComplete(clientInfo);
+#if CHIP_CONFIG_ENABLE_READ_CLIENT
```

Comment:
This file is not including the right headers to pick up this config value, so presumably it's getting it by accident. Please include the right config header.

Reply:
removed, thanks